Libraries/CameraRoll/RCTImagePickerManager.m

```diff
@@ -65,7 +65,7 @@ - (dispatch_queue_t)methodQueue
   imagePicker.sourceType = UIImagePickerControllerSourceTypeCamera;
 
   if ([RCTConvert BOOL:config[@"videoMode"]]) {
-    imagePicker.cameraCaptureMode = UIImagePickerControllerCameraCaptureModeVideo;
+    imagePicker.mediaTypes = @[(NSString *)kUTTypeImage, (NSString *)kUTTypeMovie];
```

Comment:
Why not just:

   @[(NSString *)kUTTypeMovie];

Since we're in video mode, why also enable the image media type?